packages/composites/augur/src/methods/createMarkets.ts

```diff
@@ -84,18 +96,34 @@ const createTeam = async (jobRunID: string, sport: string, contractAddress: stri
   let nonce = await config.wallet.getTransactionCount()
   for (let i = 0; i < events.length; i++) {
     const event = events[i]
-    const payload = [
-      event.id,
-      event.homeTeamId,
-      event.awayTeamId,
-      Math.floor(event.startTime / 1000),
-      Math.round(event.homeSpread*10),
-      Math.round(event.totalScore*10),
-      event.createSpread,
-      event.createTotalScore,
-      event.moneylines,
-      { nonce }
-    ]
+    const payload =
+      sport !== 'nfl'
```

Comment:
Above we're checking if === NFL, while here we're checking !== NFL. Could we be consistent with checking the same thing on both, so it's easier to follow the flow?